internal/gatewayapi/testdata/gateway-with-listener-with-tcproute-with-mismatch-port-protocol.out.yaml

```diff
@@ -94,3 +94,10 @@ xdsIR:
     - address: 0.0.0.0
       name: envoy-gateway/gateway-1/tcp
       port: 10162
+      routes:
+      - destination:
+          name: tcproute/default/tcproute-1/rule/-1
+          settings:
+          - weight: 1
+        name: tcproute/default/tcproute-1
+        tls: {}
```

Comment:
curious why we have an empty `tls` struct here

Reply:
see https://github.com/envoyproxy/gateway/pull/3212/commits/9084108b4c3ac8736a0a17a9764f534029e71261